flytekit/clients/raw.py

```diff
@@ -110,6 +112,24 @@ def __init__(self, cfg: PlatformConfig, **kwargs):
         self._cfg = cfg
         if cfg.insecure:
             self._channel = grpc.insecure_channel(cfg.endpoint, **kwargs)
+        elif cfg.insecure_skip_verify:
+            # Get port from endpoint or use 443
+            try:
+                port = int(cfg.endpoint.rsplit(":", 1)[1])
+            except (IndexError, ValueError):
+                port = 443
+            cert = ssl.get_server_certificate((cfg.endpoint, port))
```

Comment:
I don't think this is correct... if `cfg.endpoint = "example.com:8443"` the pair generated here will be `("example.com:8443", 8443)` whereas it should have been `("example.com", "8443")` ... notice two differences here:
1. The port should continue to be a string per [docs](https://pythontic.com/ssl/ssl-module/get_server_certificate)
2. The first element of the tuple (pair) should NOT contain the port anymore... 

The actual implementation of the API could be more lax but I would rather stick to the official docs in case the implementation catches on later...